### PR TITLE
were drastically changed with R\&D from February 2015 until May 4 201…

### PR DESCRIPTION
**Pull request by jamesray1**
…5 (\cite{commitdateforEthash})

### CODE REVIEW

**Comment by pirapira** [comment]
This content looks like annotators of YP, not for the YP itself.

**Comment by jamesray1** [comment]
It is now inaccurate to say "have been drastically changed in the last month of research and development". It is OK to make publisher's notes or edits to keep material relevant and up-to-date. I disagree about you closing this on this premise.

**Comment by pirapira** [comment]
You're right.

**Comment by jamesray1** [comment]
Fixed merge conflicts.

File: Biblio.bib [hunk 1]
```diff
@@ -1,3 +1,13 @@
+
+@misc{commitdateforEthash,
+	url = "https://github.com/ethereum/yellowpaper/commit/77a8cf2428ce245bf6e2c39c5e652ba58a278666#commitcomment-24644869",
+	note = {{Last modified on 4 May 2015. Not able to be archived by the Wayback Machine, since the saved page doesn't load showing the cited change and comment. The change is available here: \url{https://drive.google.com/open?id=0B6erNjejWCBwajJLR2FnRWotTFk} in this folder of screenshots of the commit the change is in here: \url{https://drive.google.com/open?id=0B6erNjejWCBwaUFISUJMWjRtRHc}}},
```
Comment (pirapira) [inline Biblio.bib]: Can we just keep the link to GitHub?

Comment (jamesray1) [reply Biblio.bib]: Fixed.

**Pull request merged**